package/Makefile.winx86

```diff
@@ -7,19 +7,19 @@ NSIS_OPTS     := /V3
 NSIS_DIR      := $(ROOT_DIR)/package/winx86
 NSIS_SCRIPT   := $(NSIS_DIR)/dronin.nsi
 NSIS_TEMPLATE := $(NSIS_DIR)/dronin.tpl
-NSIS_HEADER   := $(BUILD_DIR)/ground/gcs/dronin.nsh
+NSIS_HEADER   := $(PACKAGE_DIR)/$(GCS_PACKAGE_NAME)/dronin.nsh
 
-standalone: 
+standalone: gcs
```

Comment:
Yah, I know it will let you do something bad / you need to be careful.  We also have dependency problems in various places in the code, so unless you do a make clean you are not guaranteed a good package build either.

The Jenkins configuration we have guarantees clean artifact builds with matching githash.  It's a much simpler configuration/packaging than before.  I'd rather keep packaging as a separate step that depends on completing a proper build before (so we can use parallelism properly in the earlier steps, etc).

One thing that might be a good idea (I'd be willing to do it) would be to export the githash as a symbol in the GCS executable-- we could assert that it's correct as part of the packaging process.